sdk/src/main/scala/com/horizen/SidechainSettings.scala

```diff
@@ -94,7 +94,8 @@ case class EthServiceSettings(
 case class AccountMempoolSettings(maxNonceGap: Int = 16,
                                   maxAccountSlots: Int = 16,
                                   maxMemPoolSlots: Int = 6144,
-                                  maxNonExecMemPoolSlots: Int = 1024){
+                                  maxNonExecMemPoolSlots: Int = 1024,
+                                  txLifetime: FiniteDuration = 3.hours){
```

Comment:
The default value in `TxCache` ctor was used just for avoiding updating the UT. I have preferred updating the `TxCache` ctor and tests rather than creating a constant.